src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -433,7 +433,7 @@ private void waitForCommentRowContentEquals(String commentRowIdSuffix, String co
     }
 
     /**
-     * Verifies the new comment is added (works on both modal and inline).
+     * Waits for the verification of new comment added (works on both modal and inline).
```

Comment:
I think adding the word **verification** doesn't add any value. Let's omit that?